edb/server/compiler/status.py

```diff
@@ -32,17 +32,17 @@ def get_status(ql: qlast.Base) -> bytes:
 
 @get_status.register(qlast.CreateObject)
 def _ddl_create(ql):
-    return b'CREATE'
+    return f'CREATE {ql.object_class}'.encode()
```

Comment:
What if `object_class` enum value contains an underscore in it?

Reply:
It's not supposed to, we use spaces there.